saleor/tests/e2e/orders/test_order_product_with_percentage_promotion.py

```diff
@@ -0,0 +1,240 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_category,
+    create_product,
+    create_product_channel_listing,
+    create_product_type,
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+from ..promotions.utils import create_promotion, create_promotion_rule
+from ..shipping_zone.utils import (
+    create_shipping_method,
+    create_shipping_method_channel_listing,
+    create_shipping_zone,
+)
+from ..utils import assign_permissions
+from ..warehouse.utils import create_warehouse, update_warehouse
+from .utils.draft_order_complete import draft_order_complete
+from .utils.draft_order_create import draft_order_create
+from .utils.draft_order_update import draft_order_update
+from .utils.order_lines import order_lines_create
```

Comment:
This import can be done like this: 
```
from .utils import (
    draft_order_complete,
    draft_order_create,
    draft_order_update,
    order_lines_create,
)
```
You need to fix this file name (missing `_` before .py) as it was incorrectly named in  previous PR https://github.com/saleor/saleor/pull/13639/files#diff-72a70600abee375d80b9ef5d4082300c42cb346e4ca629e727f5930b553fcada
<img width="578" alt="image" src="https://github.com/saleor/saleor/assets/30683248/a25f3ea4-7577-403b-8a50-ac591aabf8a1">

Reply:
https://github.com/saleor/saleor/pull/13702/commits/c4716739b4a4840d9ed3b91093ed24e5ae86d719